ci/vizier_build_release.sh

```diff
@@ -41,8 +41,44 @@ fi
 output_path="gs://${bucket}/vizier/${release_tag}"
 latest_output_path="gs://${bucket}/vizier/latest"
 
-bazel run --stamp -c opt --//k8s:image_version="${release_tag}" \
-    --stamp "${build_type}" //k8s/vizier:vizier_images_push "${extra_bazel_args[@]}"
+push_images_for_arch() {
+  arch="$1"
+  bazel run --stamp -c opt --//k8s:image_version="${release_tag}-${arch}" \
+      --config="${arch}_sysroot" \
+      --stamp "${build_type}" //k8s/vizier:vizier_images_push "${extra_bazel_args[@]}" > /dev/null
+  bazel run --stamp -c opt --//k8s:image_version="${release_tag}-${arch}" \
+      --config="${arch}_sysroot" \
+      --stamp "${build_type}" //k8s/vizier:list_image_bundle "${extra_bazel_args[@]}"
+}
+
+x86_64_image_list="$(mktemp)"
+aarch64_image_list="$(mktemp)"
+push_images_for_arch "x86_64" > "${x86_64_image_list}"
+push_images_for_arch "aarch64" > "${aarch64_image_list}"
+
+push_multiarch_image() {
+  multiarch_image="${1/:*/:${release_tag}}"
+  echo "Building ${multiarch_image} manifest"
+  # If the multiarch manifest list already exists locally, remove it before building a new one.
+  docker manifest rm "${multiarch_image}" || true
+  docker manifest create "${multiarch_image}" "$@"
+  docker manifest push "${multiarch_image}"
+}
+
+push_all_multiarch_images() {
+  combined_image_lists="$(paste -d' ' "$@")"
+
+  while read -r -a images;
+  do
+    push_multiarch_image "${images[@]}"
+  done < <(echo "${combined_image_lists}")
```

Comment:
This relies on both lists being the same order. Instead let's create a list of images to push with the release tag (and no arch suffix) and just use that as the input to `push_multiarch_image`
We should just assume that images with all arch suffixes exist when trying to push and can error if something is missing.